toolbox/__init__.py

```diff
@@ -149,7 +157,11 @@ def load_from_browser(self, fpath=None):
         else:
             name = fpath.name
             speaker_name = fpath.parent.name
-        
+
+        if (str(fpath)[-3:] == "mp3"):
+                print("Error: No mp3 file argument was passed but an mp3 file was used")
+                exit(-1)
```

Comment:
When this happens, let's do nothing instead of exiting
```suggestion
                return
```